src/features/game/expansion/components/resources/Stone.tsx

```diff
@@ -180,42 +186,44 @@ export const Stone: React.FC<Props> = ({ rockIndex, expansionIndex }) => {
   const timeLeft = getTimeLeft(rock.stone.minedAt, STONE_RECOVERY_TIME);
 
   return (
-    <div className="relative z-10 group" ref={containerRef}>
+    <div className="relative z-10 group w-full h-full" ref={containerRef}>
       <img
         src={hitbox}
         style={{
           width: `${22 * PIXEL_SCALE}px`,
           opacity: 0.3,
+          left: "50%",
+          top: "50%",
+          transform: "translate(-50%, -50%)",
         }}
-        className="z-10  cursor-pointer"
+        className="absolute z-10 cursor-pointer"
         onMouseEnter={handleHover}
         onMouseLeave={handleMouseLeave}
         onClick={strike}
       />
 
-      <div
-        className="absolute pointer-events-none"
-        style={{
-          // The GIFs are larger than the rock due to the animations, so place correctly
-          transform: `translateY(${-GRID_WIDTH_PX * 2.947}px) translateX(${
-            -GRID_WIDTH_PX * 0.6888
-          }px)`,
-        }}
-      >
+      <div className="absolute z-20 pointer-events-none w-full h-full">
         {!mined && (
           <>
             <Spritesheet
-              className="group-hover:img-highlight pointer-events-none "
+              className="relative group-hover:img-highlight pointer-events-none w-full h-full"
               style={{
-                width: `${PIXEL_SCALE * 91}px`,
+                width: `${SPARK_SHEET_FRAME_WIDTH * PIXEL_SCALE}px`,
+                height: `${SPARK_SHEET_FRAME_WIDTH * PIXEL_SCALE}px`,
                 imageRendering: "pixelated",
+                position: "absolute",
+
+                // Adjust the base of stone to be perfectly aligned
+                // between two grid boxes
+                bottom: `-${18 * PIXEL_SCALE}px`,
+                left: `-${6 * PIXEL_SCALE}px`,
```

Comment:
Positioning the stone using the pixel scale allows us to make 1 pixel changes to the placement.

![stone](https://user-images.githubusercontent.com/41215134/184061644-77016f47-0323-41af-a139-9871f88bfef4.png)